patsy/highlevel.py

```diff
@@ -33,7 +33,7 @@
 def _try_incr_builders(formula_like, data_iter_maker, eval_env,
                        NA_action):
     if isinstance(formula_like, DesignMatrixBuilder):
-        return (design_matrix_builders([[]], data_iter_maker, NA_action)[0],
+        return (design_matrix_builders([[]], data_iter_maker, eval_env, NA_action)[0],
```

Comment:
86 chars, of which I've seen a couple other ones in the code base. Would you rather have me leave it as is, or split the arguments of the `design_matrix_builders()` function call each on its own line?